engine/runtime/src/main/scala/org/enso/compiler/codegen/IrToTruffle.scala

```diff
@@ -1317,6 +1317,9 @@ class IrToTruffle(
     def processApplication(application: IR.Application): RuntimeExpression =
       application match {
         case IR.Application.Prefix(fn, args, hasDefaultsSuspended, loc, _, _) =>
+          if (args.isEmpty && hasDefaultsSuspended) {
```

Comment:
I don't understand this code much to get implications of this change. Out of curiosity: What running a `fn` means?

Reply:
This is a recursive AST builder – `run(fn)` runs the builder on `fn`.

Reply:
More deeply – if an application has defaults suspended (i.e. `...` – "don't call me") is used, and there are no arguments to curry, the whole thing is a no-op.